src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs

```diff
@@ -216,7 +216,9 @@ private void AddStyle_SaveButton_Click(object sender, RoutedEventArgs e)
 
             var colorHexString = grid.FindName("colorHexVal") as Label;
 
-            var newItem = new StyleItem() { Name = groupNameLabel.Text, HexColorString = colorHexString.Content.ToString() };
+            var groupStyleFontSize = grid.FindName("groupStyleFontSize") as ComboBox;
```

Comment:
Is the grid.FindName("groupStyleFontSize") necessary?

Reply:
Yes, in order to get the font size selected from the user, the same approach to get the name and color from the other controls.